terraform/modules/fargate/monitoring.tf

```diff
@@ -80,7 +80,11 @@ resource "aws_cloudwatch_metric_alarm" "healthy_host_low_alb" {
   statistic           = "Minimum"
   threshold           = var.desired_count
 
-  alarm_description  = "There are fewer healthy hosts than expected"
+  alarm_description  = <<EOF
+There are fewer healthy ECS tasks than expected in the ${aws_ecs_cluster.main.name} cluster.
```

Comment:
Does healthy hosts = healthy tasks?

Reply:
Yeah, I normally say "hosts" to mean EC2 instances and "tasks" to mean ECS tasks, but "host" is more general really.

This alarm is based on the load balancer health checking its targets and it doesn't know what those targets are, HealthyHostCount is a count of how many healthy targets it has.

Reply:
Slightly confused by the fact that ECS can do it's own healtchecks that we don't use